src/Open3D/Visualization/Rendering/Filament/FilamentResourceManager.cpp

```diff
@@ -57,7 +58,7 @@ Handle RegisterResource(filament::Engine& engine,
                         ResourceType* resource,
                         ResourcesContainer<ResourceType>& container) {
     if (!resource) {
-        // TODO: assert
+        utility::LogError("Trying to register empty resource!");
```

Comment:
Just fyi, utility::LogError() throws std::runtime_error, it doesn't assert, which doesn't look like what you are expecting, given the error recovery. (Took me by surprise when I discovered this, too.)

Reply:
I think, I'll leave it here, in ResourceManager, as it is. But I replaced some of occurrences in other files, because severity of errors there not so big.